modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/ModalMoveObjectDefinition.tsx

```diff
@@ -106,6 +106,9 @@ export function ModalMoveObjectDefinition({
 			if (onAfterMoveObjectDefinition) {
 				onAfterMoveObjectDefinition();
 			}
+			else {
+				setTimeout(() => window.location.reload(), 1000);
+			}
```

Comment:
Hey @rebsilva, This code is unnecessary, as all applications of the move modal no longer use loading, this `else` will never be called. If you are worried about future people who may use this modal elsewhere, to avoid any problems we can make `onAfterMoveObjectDefinition` a `required` property and remove the if above, what do you think?

```suggestion
			onAfterMoveObjectDefinition();
```

Reply:
good point @danvb112, i will change the code and send it again, thanks!